server/server.go

```diff
@@ -293,7 +295,9 @@ func (s *Server) SendChatMessage(friendDisplayName, message string) error {
 	(*chatMsg).Text = cyp[0]
 
 	chatMsg.NewPayload(s.User.MAC, s.User.IP, s.User.Username, friend.Username)
-	(*chatMsg).Text = []byte(message)
+	bytes := []byte(message)
+	(*chatMsg).Text = bytes
```

Comment:
Since I made a comment already, can you just do this in one line?

Reply:
I would rather not i use it twice and its kindof ugly

Reply:
Fine.